pkg/operator/operator.go

```diff
@@ -179,34 +121,22 @@ func New(config operatorconfig.Config, dnsManager dns.Manager, kubeConfig *rest.
 func (o *Operator) Start(stop <-chan struct{}) error {
 	// Periodicaly ensure the default controller exists.
 	go wait.Until(func() {
+		if !o.manager.GetCache().WaitForCacheSync(stop) {
+			log.Error(nil, "failed to sync cache before ensuring default ingresscontroller")
+			return
+		}
```

Comment:
Is `ensureDefaultIngressController` actually using the cache?

Reply:
Not right now, but generally it seemed like a good idea to wait until things were up and running

Reply:
Since the `wait.Until` uses a 1-minute period, does this mean that the operator will probably take an extra minute to create the default ingress controller on a new cluster?

Reply:
Hm, I guess in the normal case, `WaitForCacheSync` will block and succeed and delay the creation exactly as long as it takes to sync the cache.